Countries Info/ViewModels/ShowAllViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ShowAllViewModel.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 11.12.2020.
+//
+
+import Foundation
+import UIKit
+
+protocol ShowAllViewModelProtocol {
+    var list: [String] { get }
+    var color: UIColor { get }
```

Comment:
Just an idea, in case we want to avoid using UIKit here, you may define some tuple with colours and give it typealias name, pass it as a param and store inside view model. Later on you can have UIColor extension which accepts this tuple inside init method.